src/lib/php/common-sysconfig.php

```diff
@@ -204,65 +204,71 @@ function Populate_sysconfig()
   $valueArray[$variable] = array("'$variable'", "null", "'$oidcPrompt'",
     strval(CONFIG_TYPE_TEXT), "'OauthSupport'", "1", "'$oidcDesc'", "null", "null");
 
+  $variable = "OidcClientIdClaim";
+  $oidcPrompt = _('OIDC Client Id Claim');
+  $oidcDesc = _('e.g. "azp"<br>Client ID claim in the decoded payload.');
+  $valueArray[$variable] = array("'$variable'", "null", "'$oidcPrompt'",
```

Comment:
Let's add a default value for this to prevent unwanted errors in deployments.
```suggestion
  $valueArray[$variable] = array("'$variable'", "'client-id'", "'$oidcPrompt'",
```